lib/modules/manager/osgi/extract.spec.ts

```diff
@@ -0,0 +1,200 @@
+import { Fixtures } from '../../../../test/fixtures';
+import { extractPackageFile } from './extract';
+
+const unsupportedFeatureVersion = Fixtures.get(
+  'unsupported-feature-version.json'
+);
+const featureWithBundlesAsObjects = Fixtures.get('bundles-as-objects.json');
```

Comment:
why these aren't inlined?

Reply:
I misread the previous comments - I thought I should've moved only "small" fixtures.

Reply:
they where all small 🙃